debian/changelog

```diff
@@ -1,3 +1,10 @@
+kolibri-server (0.5.0-0ubuntu2) jammy; urgency=medium
```

Comment:
I don't know what the versioning protocol is for this

Reply:
I think the post-release numbers are only used for packaging issues? I think we can just call it `0.5.1-0ubuntu1`.

Reply:
I have found a very thorough guide on debian package versioning: https://serverfault.com/a/604549

I am still not sure I understand though!